crates/rpc/rpc-api/src/engine.rs

```diff
@@ -105,6 +117,12 @@ pub trait EngineApi {
     ) -> RpcResult<ExecutionPayloadBodiesV1>;
 
     /// See also <https://github.com/ethereum/execution-apis/blob/6709c2a795b707202e93c4f2867fa0bf2640a84f/src/engine/paris.md#engine_exchangetransitionconfigurationv1>
+    ///
+    /// Note: This method will be deprecated after the cancun hardfork:
+    ///
+    /// > Consensus and execution layer clients MAY remove support of this method after Cancun. If
```

Comment:
can we create a separate issue for cleaning up capabilities, so we don't forget?